src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Cms.cs

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class Crypto
+    {
+        [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_CmsDecode")]
+        internal static extern SafeCmsHandle CmsDecode([In] byte[] buf, int len);
+
+        [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_CmsDestroy")]
+        internal static extern void CmsDestroy(IntPtr cms);
+
+        [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_CmsGetEmbeddedContent")]
```

Comment:
You can leave off the EntryPoint value when it matches the name of the extern method.

It doesn't really hurt to have it, so really it's just a stylistic thing.